quizcon/templates/courseaffils/course_detail_student.html

```diff
@@ -1,6 +1,10 @@
-<div class="row pt-3 px-0 mt-0 mx-0">
-    <div class="col-12">
-        {{course.title}}
-        <div><b>Your role:</b> {% if is_faculty %}Instructor{% else %}Student{% endif %}</div>
+<div class="row justify-content-md-center">
+    <div class="col-md-auto text-center my-5">
+        <h2>403: Forbidden</h2>
+
+        <p>
+          We're sorry. This content is not available for students.
```

Comment:
A user who is not a class member will also hit this error message. Do we want to broaden the message?